pom.xml

```diff
@@ -5,9 +5,17 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>80</version>
```

Comment:
it looks like you depends on other PR which is in progress, it should be a separate commit (cherry-picked)

Reply:
Please see:
https://github.com/prestodb/presto/pull/11166#discussion_r206802469

It is hard for me to locally separate out this commit, and my local testing needs these changes. Hence I raised this as part of the PR, will revert before merging. Please ignore any changes to this file

Reply:
Please see:
https://github.com/prestodb/presto/pull/11166#discussion_r206802469

It is hard for me to locally separate out this commit, and my local testing needs these changes. Hence I raised this as part of the PR, will revert before merging. Please ignore any changes to this file